composer.json

```diff
@@ -11,13 +11,9 @@
     ],
     "require": {
         "php": "^8.1",
-        "ext-hash": "*",
-        "ext-json": "*",
         "ext-mongodb": "^1.21.0",
         "composer-runtime-api": "^2.0",
-        "psr/log": "^1.1.4|^2|^3",
-        "symfony/polyfill-php80": "^1.27",
-        "symfony/polyfill-php81": "^1.27"
```

Comment:
These were also removed in https://github.com/mongodb/mongo-php-library/pull/1619, but there doesn't appear to be any conflict here.